framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/AzureQueueStorageChannelHelpers.cs

```diff
@@ -0,0 +1,124 @@
+            {
+                return messageEncoderFactory.MessageVersion;
+            }
+        }
+
+        // we can use the same encoder for all our Udp Channels as it's free-threaded
+        internal static MessageEncoderFactory DefaultMessageEncoderFactory
+       {
+            get
+            {
+                return messageEncoderFactory;
+            }
+        }
+    }
+
+    static class UdpConfigurationStrings
+    {
+        public const string MaxBufferPoolSize = "maxBufferPoolSize";
+        public const string MaxReceivedMessageSize = "maxMessageSize";
+        public const string Multicast = "multicast";
+        public const string OrderedSession = "orderedSession";
+        public const string ReliableSessionEnabled = "reliableSessionEnabled";
+        public const string SessionInactivityTimeout = "sessionInactivityTimeout";
+        public const string ClientBaseAddress = "clientBaseAddress";
+    }
+
+    static class UdpPolicyStrings
+    {
+        public const string UdpNamespace = "http://sample.schemas.microsoft.com/policy/aqs";
+        public const string Prefix = "aqs";
+        public const string MulticastAssertion = "Multicast";
+        public const string TransportAssertion = "soap.aqs";
+    }
+
+    static class AzureQueueStorageChannelHelpers
+    {
+        /// <summary>
+        /// The Channel layer normalizes exceptions thrown by the underlying networking implementations
+        /// into subclasses of CommunicationException, so that Channels can be used polymorphically from
+        /// an exception handling perspective.
+        /// </summary>
+        internal static CommunicationException ConvertTransferException(SocketException socketException)
+        {
+            return new CommunicationException(
+                string.Format(CultureInfo.CurrentCulture, 
+                "A Udp error ({0}: {1}) occurred while transmitting data.", socketException.ErrorCode, socketException.Message), 
+                socketException);
+        }
+
+        internal static bool IsInMulticastRange(IPAddress address)
+        {
+            if (address.AddressFamily == AddressFamily.InterNetwork)
+            {
+                // 224.0.0.0 through 239.255.255.255
+                byte[] addressBytes = address.GetAddressBytes();
+                return ((addressBytes[0] & 0xE0) == 0xE0);
+                //(address.Address & MulticastIPAddress.IPv4MulticastMask) == MulticastIPAddress.IPv4MulticastMask);
+            }
+            else
+            {
+                return address.IsIPv6Multicast;
+            }
+        }
+
+        internal static void ValidateTimeout(TimeSpan timeout)
+        {
+            if (timeout < TimeSpan.Zero)
+            {
+                throw new ArgumentOutOfRangeException("timeout", timeout, "Timeout must be greater than or equal to TimeSpan.Zero. To disable timeout, specify TimeSpan.MaxValue.");
+            }
+        }
+    }
+
+    static class UdpDefaults
+    {
```

Comment:
Is this still referenced? If so, rename, otherwise remove.

Reply:
It was being used to for setting up the Multicast variable. Since that's no longer needed, I am removing this as well.